packages/runtime/container-runtime/src/gc/garbageCollection.md

```diff
@@ -50,16 +50,18 @@ Mark phase is enabled by default for a container. It is enabled during creation
 
 ### Sweep phase
 
-In this phase, the GC algorithm identifies all Fluid objects that have been unreferenced for a specific amount of time (typically 30-40 days) and deletes them.
-Objects are only swept once the GC system is sure that they could never be referenced again by any active clients, i.e., clients that have the object in memory and could reference it.
-The Fluid Runtime enforces a maximum session length (configurable) in order to guarantee an object is safe to delete after sufficient time has elapsed.
-
-GC sweep phase has not been enabled by default yet. A "soft" version of Sweep called "Tombstone Mode" is enabled by default
-as part of the Mark Phase when Sweep is disabled. In this mode, any object that GC determines is ready to be deleted is
-marked as a "Tombstone", which triggers certain logging events and/or behavior changes if/when that Tombstoned object is
-accessed by the application.
-
-Tombstone is intended for use by early adopters of GC and is documented in more detail [here](./gcEarlyAdoption.md).
+In this phase, the GC algorithm deletes any Fluid object that has been unreferenced for a sufficient time to guarantee
```

Comment:
Should we add Tombstone as a separate phase rather than making it part of the sweep phase? Just a thought, I am not too sure about it.

Reply:
Thought about it. I think if/when we put a gate at Tombstone to ensure all objects pass through Tombstone (and stay there for the grace period), we should update the language to 3-phase.